README.md

```diff
@@ -42,17 +42,50 @@ The branch `compose-latest` is targetting the latest alpha versions of Compose.
+
+## High level features 
+
+Looking for a sample that has the following features?
+
+### Custom Layouts
+* [Jetnews: Interests Screen](https://github.com/android/compose-samples/blob/ee198110d8a7575da281de9bd0f84e91970468ca/JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt#L428)
+* [Jetchat: AnimatedFabContent](https://github.com/android/compose-samples/blob/ee198110d8a7575da281de9bd0f84e91970468ca/Jetchat/app/src/main/java/com/example/compose/jetchat/components/AnimatingFabContent.kt#L101)
+* [Jetsnack: Grid](https://github.com/android/compose-samples/blob/73d7f25815e6936e0e815ce975905a6f10744c36/Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Grid.kt#L27)
+* [Owl: Staggered Grid](https://github.com/android/compose-samples/blob/ee198110d8a7575da281de9bd0f84e91970468ca/Owl/app/src/main/java/com/example/owl/ui/courses/FeaturedCourses.kt#L67)
+
+### Theming
+* [Jetchat: Material3](https://github.com/android/compose-samples/blob/main/Jetchat/app/src/main/java/com/example/compose/jetchat/theme/Themes.kt#L91)
+* [Jetcaster: Custom theme based on cover art](https://github.com/android/compose-samples/blob/main/Jetcaster/app/src/main/java/com/example/jetcaster/util/DynamicTheming.kt)
+* [Jetsnack: Custom Design System](https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/theme/Theme.kt)
+
+### Animations
+* [Crane: Calendar Selection Animation](https://github.com/android/compose-samples/blob/87ab51822cd87ce07b8ea3ec9dcb8b93f04c0b2a/Crane/app/src/main/java/androidx/compose/samples/crane/calendar/Calendar.kt#L64)
+* [Crane: AnimatedContent](https://github.com/android/compose-samples/pull/843)
+* [Jetsurvey: AnimatedContent](https://github.com/android/compose-samples/pull/842)
+* [Jetcaster: Animated theme colors](https://github.com/android/compose-samples/blob/69e9d862b5ffb321064364d7883e859db6daeccd/Jetcaster/app/src/main/java/com/example/jetcaster/util/DynamicTheming.kt)
+* [Owl: Transition - Topic Chip change](https://github.com/android/compose-samples/blob/5dd149f2a5d35527cb628972c6c058057233647f/Owl/app/src/main/java/com/example/owl/ui/onboarding/Onboarding.kt#L181)
+* [Jetsnack: Parallax Scroll](https://github.com/android/compose-samples/blob/e9f5d03ccc7de29165853914749fcf7af70de823/Jetsnack/app/src/main/java/com/example/jetsnack/ui/snackdetail/SnackDetail.kt)
```

Comment:
Just saw this after leaving the comment above. The Jetsnack readme calls this a CollapsingImageLayout. Might make more sense in the custom layouts section but up to you.

Jetsnack also has a custom animating bottom bar, you could replace this one with that. https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

Reply:
```suggestion
* [Jetsnack: Animating Bottom Barl](https://github.com/android/compose-samples/blob/main/Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt)
```